libs/auth-api-lib/src/lib/entities/models/personal-representative.model.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  Column,
+  CreatedAt,
+  UpdatedAt,
+  DataType,
+  Model,
+  Table,
+  PrimaryKey,
+  HasMany,
+} from 'sequelize-typescript'
+import { ApiProperty } from '@nestjs/swagger'
+import { PersonalRepresentativeRight } from './personal-representative-right.model'
+import { PersonalRepresentativeDTO } from '../dto/personal-representative.dto'
+
+@Table({
+  tableName: 'personal_representative',
+})
+export class PersonalRepresentative extends Model<PersonalRepresentative> {
+  @PrimaryKey
+  @Column({
+    type: DataType.STRING,
```

Comment:
Could let the database generate UUIDs:

@Column({
    type: DataType.UUID,
    primaryKey: true,
    allowNull: false,
    defaultValue: DataType.UUIDV4,
  })
  @ApiProperty()
  id!: string

Reply:
Will explore in next PR.